client/src/telethon/_impl/client/types/keyboard.py

```diff
@@ -55,4 +55,24 @@ def __init__(
         self._raw = types.ReplyInlineMarkup(rows=_build_keyboard_rows(buttons))
 
 
-KeyboardType: TypeAlias = Keyboard | InlineKeyboard
+class HiddenKeyboard:
+    __slots__ = ("_raw",)
+
+    def __init__(self, selective: bool) -> None:
```

Comment:
Personally a big fan of forcing named arguments. `HiddenKeyboard(False)` reads poorly.